src/platform/silabs/platformAbstraction/GsdkSpam.cpp

```diff
@@ -40,7 +67,25 @@ SilabsPlatform::SilabsButtonCb SilabsPlatform::mButtonCallback = nullptr;
 
 CHIP_ERROR SilabsPlatform::Init(void)
 {
-    init_efrPlatform();
+    sl_system_init();
+    sl_mbedtls_init();
+
+#if CHIP_ENABLE_OPENTHREAD
+    sl_ot_sys_init();
```

Comment:
Please test with the thread libs if you didn't already. I recall having an issue with this function call and had to split it. Maybe it is fixed now but let's just confirm.

The previous init_efrplatform also had. Ifdef for mgm24. Those that work for that family too?

Reply:
Already did and it was able to commission with Google.

Reply:
This function is generic and works with all boards provided that the new lib is present